lib/matplotlib/animation.py

```diff
@@ -1585,6 +1585,10 @@ def new_saved_frame_seq(self):
         # Generate an iterator for the sequence of saved data. If there are
         # no saved frames, generate a new frame sequence and take the first
         # save_count entries in it.
+        # itertools.islice can return an error when passed a numpy int instead
+        # of a native python int. This is a known issue:http://bugs.python.org/issue30537
+        # As a workaround, enforce conversion to native python int
+        self.save_count = int(self.save_count)
```

Comment:
I would do the conversion in `__init__` (after the check for None).  Also needs a space after the colon in the comment just above.